docs/pages/installation/Customization.vue

```diff
@@ -19,7 +19,7 @@
                 <h3 class="subtitle">Set your variables with Sass</h3>
                 <b-message type="is-info">
                     <div class="content">
-                        <p><b>Tip:</b> You can see <a href="http://bulma.io/documentation/overview/variables/" target="_blank">all of Bulma's variables</a>.</p>
+                        <p><b>Tip:</b> You can see <a href="https://web.archive.org/web/20240107060345/https://bulma.io/documentation/overview/variables/" target="_blank">all of Bulma's variables</a>.</p>
```

Comment:
I believe this would be the appropriate link to use https://versions.bulma.io/0.9.4/documentation/customize/variables/

Reply:
You are right, I updated the Pull Request